emmet-core/emmet/core/electrode.py

```diff
@@ -0,0 +1,222 @@
+from datetime import datetime
+from typing import Dict, List
+
+from monty.json import MontyDecoder
+from pydantic import BaseModel, Field, validator
+from pymatgen.apps.battery.battery_abc import AbstractElectrode
+from pymatgen.apps.battery.conversion_battery import ConversionElectrode
+from pymatgen.apps.battery.insertion_battery import InsertionElectrode
+from pymatgen.core.periodic_table import ElementBase
+from pymatgen.entries.computed_entries import ComputedEntry
+
+from emmet.stubs import Composition, Structure
+
+
+class WorkingIon(ElementBase):
+    Li = "Li"
+    Ca = "Ca"
+    Mg = "Mg"
+
```

Comment:
This can just be pymatgen `Element`.

Reply:
So I did this with the API in mind since this can be used directly for the drop down ion selection.

Reply:
I would put that at the API level and not limit it here. The filters are separate from the doc model.